modules/rdma-configuring-the-nvidia-network-operator.adoc

```diff
@@ -0,0 +1,230 @@
+        enable: true
+        force: true
+        timeoutSeconds: 300
+        podSelector: ''
+      maxParallelUpgrades: 1
+      safeLoad: false
+      waitForCompletion:
+        timeoutSeconds: 0
+    startupProbe:
+      initialDelaySeconds: 10
+      periodSeconds: 20
+    image: doca-driver
+    repository: nvcr.io/nvidia/mellanox
+    version: 24.10-0.7.0.0-0
+    env:
+    - name: UNLOAD_STORAGE_MODULES
+      value: "true"
+    - name: RESTORE_DRIVER_ON_POD_TERMINATION
+      value: "true"
+    - name: CREATE_IFNAMES_UDEV
+      value: "true"
+----
+
+. Create the `NicClusterPolicy` custom resource on the cluster:
++
+[source,terminal]
+----
+$ oc create -f network-sharedrdma-nic-cluster-policy.yaml 
+----
++
+.Example output
+[source,terminal]
+----
+nicclusterpolicy.mellanox.com/nic-cluster-policy created
+----
+
+. Validate the `NicClusterPolicy` by running a few commands in the DOCA/MOFED container:
++
+[source,terminal]
+----
+$ oc get pods -n nvidia-network-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                          READY   STATUS    RESTARTS   AGE
+doca-telemetry-service-hwj65                                  1/1     Running   2          160m
+kube-ipoib-cni-ds-fsn8g                                       1/1     Running   2          160m
+mofed-rhcos4.16-9b5ddf4c6-ds-ct2h5                            2/2     Running   4          160m
+nic-feature-discovery-ds-dtksz                                1/1     Running   2          160m
+nv-ipam-controller-854585f594-c5jpp                           1/1     Running   2          160m
+nv-ipam-controller-854585f594-xrnp5                           1/1     Running   2          160m
+nv-ipam-node-xqttl                                            1/1     Running   2          160m
+nvidia-network-operator-controller-manager-5798b564cd-5cq99   1/1     Running   2         5d23h
+rdma-shared-dp-ds-p9vvg                                       1/1     Running   0          85m
+----
+
+. You can `rsh` into the `mofed` container to check the status:
++
+[source,terminal]
+----
+$ MOFED_POD=$(oc get pods -n nvidia-network-operator -o name | grep mofed)
+$ oc rsh -n nvidia-network-operator -c mofed-container ${MOFED_POD}
+sh-5.1# ofed_info -s
+OFED-internal-24.07-0.6.1:
+sh-5.1# ibdev2netdev -v
+0000:0d:00.0 mlx5_0 (MT41692 - 900-9D3B4-00EN-EA0) BlueField-3 E-series SuperNIC 400GbE/NDR single port QSFP112, PCIe Gen5.0 x16 FHHL, Crypto Enabled, 16GB DDR5, BMC, Tall Bracket                                                       fw 32.42.1000 port 1 (ACTIVE) ==> ibs2f0 (Up)
+0000:a0:00.0 mlx5_1 (MT41692 - 900-9D3B4-00EN-EA0) BlueField-3 E-series SuperNIC 400GbE/NDR single port QSFP112, PCIe Gen5.0 x16 FHHL, Crypto Enabled, 16GB DDR5, BMC, Tall Bracket                                                       fw 32.42.1000 port 1 (ACTIVE) ==> ens8f0np0 (Up)
+----
+
+. Create a `IPoIBNetwork` custom resource file:
++
+[source,terminal]
```

Comment:
Same thing about making this YAML:
```suggestion
[source,yaml]
```

Please apply this feedback globally

Reply:
Changed.